.github/workflows/release-build-publish.yml

```diff
@@ -37,3 +37,5 @@ jobs:
         env:
           GITHUB_ACTOR: ${{ github.actor }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  # TODO: On release, publish the Kotlin classes & the p8e contracts
```

Comment:
Im happy to come update the build again once you have everything you'd like in place :D